src/map/status.c

```diff
@@ -7395,6 +7395,9 @@ static int status_change_start_sub(struct block_list *src, struct block_list *bl
 			//Undead are immune to Freeze/Stone
 			if (undead_flag && !(flag&SCFLAG_NOAVOID))
 				return 0;
+			// If player uner lex while getting stonned/frozen it shall be removed
```

Comment:
Typos in comment.
-> SC_LEXAETERNA should be removed when applying SC_STONE or SC_FREEZE.